### DISCUSSION TITLE
Gaps in corners of Disclosure heading

### DISCUSSION THREAD

**Discussion by kyle-clark1824** [twilio-labs/paste#3251] [Q&A] [answered]
Hey team. I'm using some Disclosures and having an issue with the default border-radius.
Looks like the DISCLOSURE_HEADINGs get a border-radius: 2px. For some reason that's causing some gaps in my corners. Bumping that radius up to 6px or so seems to do the trick but it doesn't look like that's an "overridable" option from the code.

I'm sure I have a conflicting element/style that's causing this, so just wondering if anyone else has seen the issue or has thoughts.

Thanks!

![image](https://github.com/twilio-labs/paste/assets/92951350/f7276205-d663-439f-b503-35d9e86939a3)

**Comment by SiTaggart** [comment]
Hey @kyle-clark1824 

Can you create a very minimal reproduction of what you're trying to do in a Code sandbox for us, please? You can use or fork our [starter template](https://codesandbox.io/s/paste-starter-template-rz6u3?file=/src/index.tsx). It will help us better visualize what is going on.

Thanks

**Reply by kyle-clark1824** [reply]
I wasn't able to recreate the issue in the starter sandbox. 

I did however see what's going on.. not WHY but one step closer. Like I said my DISCLOSURE_HEADING gets a `border-radius: 2px`. But the containing DISCLOSURE element has a `border-radius: 8px`. This "miss match" is causing "gaps" in the corners. Changing either value to match the other in dev-tools fixes the "gaps". 

I'm not sure what leads to those values being set... but them being different seems to be my problem. 

![image](https://github.com/twilio-labs/paste/assets/92951350/411d4a1e-e6ed-48e6-b751-7f6e84ac415d)

**Reply by SiTaggart** [reply]
It's going to be super hard to figure out what is happening without being able to see what you're trying to do.

If you've recreated it in a sandbox that's a good start to give us, at least to just see the code. The fact you can't reproduce might also be a hint to the problem. For example you're not using the latest version of Paste, so an upgrade might be the solution.

Lets start with the update of Paste to v19.2.0 and go from there.

**Comment by kyle-clark1824** [comment]
This is for console zen so I'm tied to version 16.0.1 by the looks of it.

I've added my code to the sandbox but things look good.  The only difference with my code is I'm creating multiple of those dynamically.. BUT I do realize the "problem" is probably something in one of those "parent" components doing the creation.. 

https://codesandbox.io/s/new-star-zd01i9?file=/src/index.tsx

**Reply by SiTaggart** [reply, marked as answer]
My hunch is it's going to be the Paste version Console is using that's the problem.
